src/categorical_algebra/DataMigrations.jl

```diff
@@ -161,6 +162,30 @@ function migrate!(X::ACSet, Y::ACSet, FOb, FHom)
   migrate!(X, Y, M)
 end
 
+function (::Type{T})(X::ACSet, F::FinFunctor) where T <: ACSet
+  Base.depwarn("Data migration constructor is deprecated, use `migrate` instead", :ACSet)
+  migrate(T, X, F)
+end
+
+function (::Type{T})(X::ACSet, FOb::AbstractDict, FHom::AbstractDict) where T <: ACSet
+  Base.depwarn("Data migration constructor is deprecated, use `migrate` instead", :ACSet)
+  migrate(T, X, FOb, FHom)
+end
+
```

Comment:
Why add new deprecated methods?

Reply:
oops that looks like a rebase error! will delete those!